yarn.lock

```diff
@@ -1834,7 +2090,7 @@ color-name@~1.1.4:
   resolved "https://registry.yarnpkg.com/color-name/-/color-name-1.1.4.tgz#c2a09a87acbde69543de6f63fa3995c826c536a2"
   integrity sha512-dOy+3AuW3a2wNbZHIuMZpTcgjGuLU/uBL/ubcZF9OXbDo8ff4O8yVp5Bf0efS8uEoYo5q4Fx7dY9OgQGXgAsQA==
 
-colors@^1.1.2, colors@^1.4.0:
+colors@^1.4.0:
```

Comment:
shouldn't this have gone away?

Reply:
> shouldn't this have gone away?

It should but unfortunately "ember-template-recast" depends on `colors`
```
yarn why v1.22.10
warning ../../package.json: No license field
[1/4] 🤔  Why do we have the module "colors"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "colors@1.4.0"
info Reasons this module exists
   - "ember-template-recast" depends on it
   - Hoisted from "ember-template-recast#colors"
info Disk size without dependencies: "116KB"
info Disk size with unique dependencies: "116KB"
info Disk size with transitive dependencies: "116KB"
info Number of shared dependencies: 0
✨  Done in 0.56s.
```